client/app/components/packages/landuse-form/primary-contact.hbs

```diff
@@ -0,0 +1,26 @@
+{{#let @form as |form|}}
+  <form.Section @title="Primary Contact">
+    <p>
+      Add all Applicants (either individuals or organizations) and other Team Members (including Land Use Consultants and Environmental Consultants, agency staff, etc.).
+    </p>
+
+    <label>
```

Comment:
Actually, here should we use the Question component that Andy wrote up?
```
   <Ui::Question as |Q|>
      <Q.Label>
     ...
```

Reply:
Yep!